.github/workflows/ci_.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency:
+  group: docusaurus-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci_build.yml
```

Comment:
nit: Since it's only used once here I question the need for a reusable workflow, but trust you have plans :)

Reply:
Will eventually like to pull in the external docs build as well